src/pages/Challenge.jsx

```diff
@@ -54,17 +50,18 @@ const Challenge = () => {
   const { testPath, challengePath } = level
 
   const [instructions] = useDynamicMarkdownImport(challengePath)
-  const { code, setCode, tests } = useDynamicTestsImport(testPath)
+  const { code, setCode, tests, setTests } = useDynamicTestsImport(testPath)
+  const { allowNext, setAllowNext } = useState(false)
```

Comment:
useState returns an Array not an object.

```javascript
const [allowNext, setAllowNext] = useState(false)
```

Reply:
done

Reply:
The same, still being an Object, please pull your changes